### ISSUE TITLE
form-control should be 100% when inline form

### ISSUE THREAD

**Issue by tmaly1980** [vitalets/x-editable#695]
Normally, I'd expect the input fields to have 100% width rather than the minimum, as this is the default within bootstrap 3 for .form-control fields. Having an inline textarea is not attractive when the textarea is not 100% width. If it needs to be width: auto for popovers, at least have a CSS rule for the inline, ie:

.editable-container.editable-inline
{
        display: block;
}

.editable-inline .editableform .form-control
{
        width: 100%;
}

Are there scenarios where this doesn't make sense?

**Comment by markkimsal** [comment]
I agree.  With inline editing, the dimension constrains are usually part of the rigging around the form control already.

I had to add the following CSS to make my situation look presentable. (textarea inline edting)

.editable-inline .editableform .form-control,
.editable-inline .editableform .control-group
{
width: 100%;
}
.editable-inline .editableform .editable-input { 
width: 100%;
padding-right: 85px;
margin-right: -85px;
}

**Comment by djtechonline** [comment]
I also agree.  I ended up combining what you guys had and changed the link appearance slightly to use native colors. Also, I had to add some styling to move the editable-clear-x button over:

```
/* x-editable fix to make 100% wide fields */
.editable-container.editable-inline{
  display: block;
}

.editable-inline .editableform .form-control,
.editable-inline .editableform .control-group
{
  width: 100%;
}
.editable-inline .editableform .editable-input {
  width: 100%;
  padding-right: 85px;
  margin-right: -85px;
}
.editable-clear-x{
  margin-right: 85px;
}
a.editable-click{ color: inherit; }
.editable-empty{ color: inherit; }
```

**Comment by javicaballero** [comment]
The editable-buttons doesn't work with these workarounds. Any ideas? Thanks
